content/en/blog/2025/expose-otel-collector-gateway-api/index.md

```diff
@@ -0,0 +1,621 @@
+  lacked standardization across implementations and also limited routing
+  capabilities that varied widely across different implementations.
+- **Role Separation:** Gateway API separates concerns:
+  - `GatewayClass`: Defines a _type_ of load balancer (e.g., Istio, GKE LB).
+    Managed by Infrastructure Admins.
+  - `Gateway`: Represents an instance of a load balancer requesting a specific
+    `GatewayClass`. Defines listeners (ports, protocols, TLS). Managed by
+    Cluster Operators. They can be shared across namespaces as well.
+  - `HTTPRoute`, `GRPCRoute`, `TCPRoute`, `TLSRoute`, etc.: Define
+    application-level routing rules, attaching to a `Gateway`. Managed by
+    Application Developers/Owners.
+- **Portability:** Standardized API definition aims for greater portability
+  across different underlying gateway/service mesh implementations.
+- **Expressiveness:** Natively supports advanced features like header
+  manipulation, traffic splitting, mTLS configuration, gRPC routing, and more.
+
+In essence, Gateway API offers a more robust and standardized model for managing
+north-south traffic compared to the traditional Ingress API.
+
+## mTLS - A brief introduction
+
+Mutual TLS (mTLS) extends standard TLS by requiring _both_ the client and server
+to present and validate certificates for mutual authentication.
+
+Standard TLS (like HTTPS on websites) verifies the _server's_ identity to the
+_client_. Mutual TLS (mTLS) goes a step further:
+
+- The client verifies the server's identity (using the server's certificate).
+- The server _also_ verifies the client's identity (using the client's
+  certificate).
+
+Mutual TLS is important because it provides strong authentication, ensures
+end-to-end encryption, and aligns with Zero Trust security principles:
+
+- Ensures _only_ trusted clients (those possessing a valid certificate signed by
+  trusted CA) can connect to your exposed service.
+
+- All communication between the authenticated client and the server (such as a
+  Gateway) is encrypted.
+
+- It supports a Zero Trust model by requiring verification from both parties,
+  never assuming trust by default.
+
+## The Scenario
+
+Here are the steps we'll follow to expose an OTel Collector deployment outside
+the cluster.
+
+1.  Deploy an OTel Collector inside Kubernetes configured with a simple
+    OTLP/gRPC receiver.
+2.  Generate a self-signed Root CA, a server certificate (for the Gateway), and
+    a client certificate (for the external client).
+3.  Configure a Kubernetes `Gateway` resource to listen on a specific port,
+    terminate TLS, and require client certificates (mTLS).
+4.  Configure a `GRPCRoute` to route incoming gRPC traffic from the `Gateway` to
+    the internal OTel Collector service.
+5.  Configure an external client (another OTel Collector) to export data via
+    OTLP/gRPC, using the client certificate and trusting the Root CA.
+
+![Scenario Diagram](ScenarioFlow.png)
+
+## Setup
```

Comment:
I don't think we need this heading. I'd remove it and bump step 1 and step 2 up to ##-level headings. Otherwise the page nav looks a little strange:

<img width="205" alt="Screenshot 2025-05-29 at 3 13 44 PM" src="https://github.com/user-attachments/assets/b0657edc-363c-4a01-8143-b3c101c73757" />

Reply:
Something like this
<img width="304" alt="image" src="https://github.com/user-attachments/assets/1da3e4eb-ada9-4a40-8a30-634a81c85641" />